src/version.h

```diff
@@ -21,7 +21,7 @@ static const int INIT_PROTO_VERSION = 209;
 static const int GETHEADERS_VERSION = 60800;
 
 //! disconnect from peers older than this proto version
-static const int MIN_PEER_PROTO_VERSION = 70900; // Only connect to v2.3 Nodes and newer
+static const int MIN_PEER_PROTO_VERSION = 71400;  // TODO (BDAP): ***** Revert to 70900
```

Comment:
70900

Reply:
That version is needed for our testnet but I have this commit to revert the version to 2.4.0 after this merge:  https://github.com/duality-solutions/Dynamic/commit/717f18d361993ecfc17415f4c8aaef2cd61c3793